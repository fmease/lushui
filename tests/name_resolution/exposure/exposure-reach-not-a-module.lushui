;; TEST fail check
;; @Task also report the undefined `xxx` if it's feasible to implement

@(public Void) data Empty: Type =

data Hunk: Type =
    hunk: Hunk

imposter: Hunk = xxx ;; Hunk.hunk

@(public imposter) data Void: Type =
