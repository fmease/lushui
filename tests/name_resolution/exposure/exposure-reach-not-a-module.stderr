error[E022]: value `Void` is not a module
 --> ${DIRECTORY}/name_resolution/exposure/exposure-reach-not-a-module.lushui:4:10
  |
4 | @(public Void) data Empty: Type of
  |          ^^^^ 
  |

error[E022]: value `imposter` is not a module
  --> ${DIRECTORY}/name_resolution/exposure/exposure-reach-not-a-module.lushui:11:10
   |
11 | @(public imposter) data Void: Type of
   |          ^^^^^^^^ 
   |

error: aborting due to 2 previous errors

